drivers/include/mrf24j40.h

```diff
@@ -125,6 +125,18 @@ extern "C" {
 #define MRF24J40_USE_EXT_PA_LNA         (0U)
 #endif
 
+/**
+ * @brief Enable basic self-test on init
+ *
+ * Perform a write / read to a known register on startup to detect
+ * if the device is connected.
+ * Enable this if you want the boot not to hang if the device is
+ * not connected / there are SPI errors.
+ */
```

Comment:
Maybe it makes sense to enclose this in config group so it is exposed as a compile-time config?

Reply:
You mean add a `@ingroup drivers_mrf24j40`?

Reply:
I mean to define a group like `drivers_mrf24j0_config` and add `@ingroup  config`, so it shows up [here](http://doc.riot-os.org/group__config.html)

Reply:
Ok, added.